internal/batches/service/workspaces.go

```diff
@@ -35,11 +35,6 @@ func findWorkspaces(
 	finder directoryFinder,
 	repos []*graphql.Repository,
 ) ([]RepoWorkspace, error) {
-	repoByID := make(map[string]*graphql.Repository)
-	for _, repo := range repos {
-		repoByID[repo.ID] = repo
-	}
```

Comment:
Fun fact: `repoByID` was never used again.

Reply:
Where you at golangci-lint when we need you 🧐